toy-submission/README.md

```diff
@@ -0,0 +1,13 @@
+### Make your GPUs visible to Docker 
+```
+nvidia-ctk runtime configure
+systemctl restart docker
+```
+
+### Build and run 
+```
+docker build -t toy_submission .
+docker run --gpus all -p 8080:80 toy_submission
```

Comment:
re-constraints, we can do to run on a single GPU with only 8 CPU cores (I'd be fine with 16 or 32 too) which is what you get on AWS for an A10G

```
docker run --gpus '"device=0"' --cpus=8 -p 8080:80 toy_submission
```

Regarding disk size we can't explicitly set the contraint from running docker so instead we can add something like

```
{
  "storage-driver": "overlay2",
  "storage-opts": [
    "overlay2.override_kernel_check=true",
    "overlay2.size=500GB"
  ]
}
```

to ` /etc/docker/daemon.json` so the container hard errors in case it builds to something larger than 500GB